pkgs/development/tools/corundum/default.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, rubyTool }:
+
+rubyTool {
+  name = "corundum-0.6.2";
```

Comment:
is it possible to just set the `pname` and have the version derived from the gemset.nix?